_data/sites.json

```diff
@@ -7540,6 +7540,18 @@
             "keepa.com"
         ]
     },
+    
+    {
+        "name": "KDE Bugtracking System",
+        "url": "https://www.reddit.com/r/kde/comments/agyksj/comment/eea3sni",
```

Comment:
I know this is the source of information, but this URL field is reserved for the URL that takes you to the deletion page, or to the website itself.

The source of the information can be on the commit or PR description.